roles/etcd/handlers/main.yml

```diff
@@ -13,19 +13,19 @@
   service:
     name: etcd
     state: restarted
-  when: is_etcd_master
+  when: ('etcd' in group_names)
```

Comment:
The parentheses are there for another reason - to "escape" the quotation (otherwise, it is interpreted as a YAML quote, and the YAML parser fails to interpret the value). 

This is the way I normally work around this "quote at start of YAML value" problem. If you know a better way to do it, I will implement it. 

I know it is still a problem because in a previous commit I forgot to add it, and all the CI tests started failing :)

Reply:
Oh that's why.
I've seen mostly using the other kind of quote for that (`"'etcd' in group_names"`) but parens look fine to me ; it can be confused precedence override, but OTOH the quotes muck up syntax highlighting, so that's a pick your poison scenario I guess.

Reply:
For me, using two kinds of quotes is a bit more unreadable - I am the kind of person that always prefers explicit parentheses to relying on precedence, so probably that is why :)